approvals/src/main/java/no/sikt/nva/approvals/domain/Approval.java

```diff
@@ -0,0 +1,49 @@
+package no.sikt.nva.approvals.domain;
+
+import java.net.URI;
+import java.util.List;
+import java.util.Objects;
+import java.util.UUID;
+import nva.commons.core.JacocoGenerated;
+
+public class Approval {
+
+    private final UUID identifier;
+    private final List<Identifier> identifiers;
+    private final URI source;
+
+    public Approval(UUID identifier, List<Identifier> identifiers, URI source) {
```

Comment:
On a more serious note: We should probably be more careful with naming here and make the difference clear between `identifier` (our internal identifier for this `Approval`) and `identifiers` (external references). Not sure what to do instead, but this could be a bit confusing?

Reply:
I agree, it will be more confusing when we will process each identifier separatlly :)
We can let it stay as is, so can we rename when we find better name for them.